src/config/config.js

```diff
@@ -4,7 +4,7 @@ require('dotenv').config()
 
 module.exports = {
 
-  port: process.env.PORT || 8000,
+  port: process.env.WASTE_PERMITS_APP_PORT || 8000,
```

Comment:
This should not be changed. Needs to be `PORT` to support Heroku. See PR #32 for more details.

Reply:
Where's that Picard facepalm emoji when you need it?? I've pushed an update now.